feature-engineering/datetime_engineering.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import numpy as np
+from typing import Callable
+
+def convert_datetime(df: pd.DataFrame, feature: str) -> pd.DataFrame:
+    """
+    Convert the specified feature to a datetime object and return modified dataframe.
+
+    Args:
+        df (pd.DataFrame): The input DataFrame containing the feature to be converted.
+        feature (str): The name of the feature to be converted.
+
+    Returns:
+        pd.DataFrame: The DataFrame with the specified feature converted to datetime.
+    """
+
+    df1 = df.copy()
+    df1[feature] = pd.to_datetime(df[feature], format = "'%Y-%m-%dT%H:%M:%S'")
+
+    return df1
+
+def create_month(df: pd.DataFrame, feature: str, name: str) -> pd.DataFrame:
+    """
+    Convert a "month" feature from the specified datetime feature and return modified dataframe.
+
+    Args:
+        df (pd.DataFrame): The input DataFrame containing the datetime feature.
+        feature (str): The name of the datetime feature.
+        name (str): The name of new "month" feature to be created.
+
+    Returns:
+        pd.DataFrame: The DataFrame including the "month" feature.
+    """
+
+    df1 = df.copy()
+    df1[feature] = pd.to_datetime(df[feature], format = "'%Y-%m-%dT%H:%M:%S'")
```

Comment:
You can assume the feature is already converted to datetime, so this isn't actually needed